### PR TITLE
Create ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57677](https://cumulocity.atlassian.net/browse/MTM-57677)
Original PR: [6060](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6060)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: fix: Data point list widget redirecting assets devices incorrectly (#5831) [GRAFT][release/cd] (#6060)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Data point list widget now redirects to the correct asset's devices
```

File: content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57677
+version: 1019.18.1
+---
+fix: Data point list widget redirecting assets devices incorrectly (#5831) [GRAFT][release/cd] (#6060)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, when clicking on a data point in the data point list widget, users were sometimes redirected to the wrong asset's devices. This issue has now been resolved. With this fix, clicking on a data point in the data point list widget will always take the user to the correct asset's devices as expected. The change improves the usability and navigation within the application.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Data point list widget now redirects to the correct asset's devices
```
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md:3]:
```suggestion
title: Data point list widget now redirects to the correct assets
```

File: content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57677
+version: 1019.18.1
+---
+In the past, when clicking on a data point in the data point list widget, users were sometimes redirected to the wrong asset's devices. This issue has now been resolved. With this fix, clicking on a data point in the data point list widget will always take the user to the correct asset's devices as expected. The change improves the usability and navigation within the application.
```
Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md:17]:
```suggestion
In the past, when clicking on the asset name in the data point list widget, users were sometimes redirected to the wrong url. This issue has now been resolved. With this fix, clicking on an asset name in the data point list widget will now redirect the user to the correct asset page.
```

File: content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Clicking on an asset name in the "Data point list" widget now redirects to the correct asset page.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md:3]:
```suggestion
title: Asset links in the "Data point list" widget now redirect to the correct URL
```

**Review by BeateRixen — APPROVED** [review]

**Review by eniosultan — APPROVED** [review]

**Pull request merged**